book-my-show/backend/java/bms-monolith/src/main/java/org/lbcc/bms/bms_monolith/repository/EventRepository.java

```diff
@@ -0,0 +1,17 @@
+package org.lbcc.bms.bms_monolith.repository;
+
+import org.lbcc.bms.bms_monolith.entity.Event;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.stereotype.Repository;
+
+import java.util.UUID;
+
+@Repository
+public interface EventRepository extends JpaRepository<Event, UUID> {
```

Comment:
Update name to IEventRepository to be consistent with rest of the code naming convention